packages/compass-aggregations/src/components/focus-mode/focus-mode.tsx

```diff
@@ -1,56 +1,151 @@
 import React from 'react';
 import {
-  InfoModal,
+  Modal,
+  ModalBody,
   Body,
+  css,
+  spacing,
+  palette,
 } from '@mongodb-js/compass-components';
 import { connect } from 'react-redux';
+import type { Document } from 'mongodb';
 
 import type { RootState } from '../../modules';
 import { focusModeDisabled } from '../../modules/focus-mode';
+import { FocusModeEditor } from './focus-mode-editor';
+import { FocusModePreview } from './focus-mode-preview';
 
+const modalStyles = css({
+  '[role="dialog"]': {
```

Comment:
In the deisgns the content of the modal stretches all the way to the bottom, in this PR there is still padding around the content in the modal